DEVELOPERS.md

```diff
@@ -35,6 +35,7 @@ DataFusion is written in Rust and it uses a standard rust toolkit:
 
 Testing setup:
```

Comment:
I wonder if we should add a short note saying "DataFusion uses the latest stable release of rust" as well?

Reply:
yes @alamb and @HaoYang670 I have added now you can check.